classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -115,7 +115,7 @@ public function post( $request ) {
 				'carrier_id'             => $label_data->label->carrier_id,
 				'service_name'           => $service_names[ $index ],
 				'status'                 => $label_data->label->status,
-				'commercial_invoice_url' => $label_data->label->commercial_invoice_url,
+				'commercial_invoice_url' => isset( $label_data->label->commercial_invoice_url ) ? $label_data->label->commercial_invoice_url : '',
```

Comment:
If `$label_data->label->commercial_invoice_url` was set explicitly to `null`, then `isset` won't catch it. If we want to be more defensive on it, we could use `isset( $label_data->label->commercial_invoice_url ) ? ... ` instead.